src/behaviours/draggable.js

```diff
@@ -254,7 +254,7 @@ export default function draggable(WrappedComponent) {
     }
 
     styles() {
-      return this.state.dragStart ? { opacity: 0, width: 0, height: 0 } : { opacity: 1 };
+      return this.state.dragStart ? { visibility: 'hidden' } : { visibility: 'initial' };
```

Comment:
Might be an interesting place to try out something like the animejs library? This makes the dragged component take up space again right?

Reply:
I'm not familiar with the animejs library, but yes, this makes a dragged component take up space.